activerecord/test/schema/schema.rb

```diff
@@ -169,7 +169,7 @@
   end
 
   create_table :comments, force: true do |t|
-    t.integer :post_id, null: false
```

Comment:
This sounds like something a test is relying on -- otherwise, why would it have been specifically set?